### ISSUE TITLE
Interrupts should default to "Use Off GCD"

### ISSUE THREAD

**Issue by ShmooDude** [Hekili/hekili#549]
Interrupts are a little wonky without this option checked.  It should probably be on by default since all interrupts are off the GCD and there's no need to sync it with the GCD on DPS abilities like Tiger's Fury/Berserk.

**Comment by Hekili** [comment]
Are you seeing something that looks like interrupts being synced to the GCD?  Can you provide a snapshot of it?

**Comment by ShmooDude** [comment]
I thought it was, but after looking close it's not.  It's the fact that the interrupt cooldown sweep resets based on abilities cast.  It makes it harder to determine when precisely to use the interrupt (from Hekili as opposed to watching the cast bar).

Example from when I was generating the snapshots you requested.

Enemy starts casting 2s casttime spell, Skull Bash starts cooldown swipe
@ about 1.5 seconds left on the spell, I use Shred and Skull Bash (in interrupt window) starts a new cooldown swipe
@ about 0.5 seconds left on the spell, I use Shred and Skull Bash does a super fast cooldown swipe.

Basically, it makes it hard to use with it resetting.  If possible, having it not reset the cooldown swipe if the target time remains the same would make it significantly more useful.  For now, I've turned on the cooldown timer on the interrupt box and made the font large so that it's less noticeable.

**Comment by Hekili** [comment]
Making the cooldown spiral overfunction to handle delays has had a bit of complexity.  I'm realizing that maybe the delay start time should be based on the start of the enemy cast, rather than the last GCD invocation or whatever else is impacting it.  So there's room to improve this, for sure.

**Comment by Hekili** [comment]
I think my new approach, using the enemy's cast start/duration rather than your interrupt's cooldown information, seems to work better.  I'm closing this ticket, but if there's more to sort out, we can reopen or start from a new ticket.
